lib/ReactViews/Story/Story.tsx

```diff
@@ -0,0 +1,321 @@
+import React, { MouseEventHandler, useEffect, useRef } from "react";
+import { withTranslation } from "react-i18next";
+import styled, { withTheme } from "styled-components";
+import { sortable } from "react-anything-sortable";
+import classNames from "classnames";
+
+import Box from "../../Styled/Box";
+import { RawButton } from "../../Styled/Button";
+import Ul from "../../Styled/List";
+import Text from "../../Styled/Text";
+import Spacing from "../../Styled/Spacing";
+import parseCustomHtmlToReact from "../Custom/parseCustomHtmlToReact";
+import Icon, { StyledIcon } from "../../Styled/Icon";
+
+export type Story = {
```

Comment:
Why use `type` here and not `interface`?

Reply:
yup will change to an interface